### DISCUSSION TITLE
Estou em dúvida sobre qual curso adquirir! DevQuest ou Origamid?

### DISCUSSION THREAD

**Discussion by hrybch** [frontendbr/forum#2409] [Discussão]
Basicamente o título.

Estou nesse dilema, me identifiquei muito com o jeito e didática dos gêmeos, mas a plataforma da Origamid parece entrelaçar um pouco o ensino da programação com UX/UI (que é minha área atual) por conta dos diversos cursos a mais deles.

Sempre curti MUITO programar, só que coisas bobas, pequenas e jogos simples também, mas até então nunca tive essa vontade terminal de aprender para trabalhar como estou agora, tô doido para pular para esse barco profissional e quero um suporte maneiro. 

Já vi que são dois cursos muito bons, mas na visão geral: qual dos dois vocês recomendariam?

**Comment by Diego-Girao** [comment]
Então, ai vai da sua identificação, eu curto e uso até hoje os conteúdos da Origamid, esse mês ou mês passado eles atualizaram o conteúdo dos cursos e removeram o que não se utiliza mais ou que deixou de ser boas práticas. #Origamid 💪🏼
